packages/web-app-files/src/components/FilesList/ListInfo.vue

```diff
@@ -37,7 +48,8 @@ export default {
   },
   computed: {
     items() {
-      return this.files + this.folders
+      const filesAndFolderCount = this.files + this.folders
+      return this.showSpaces ? filesAndFolderCount + this.spaces : filesAndFolderCount
```

Comment:
Why not single line as follows?
```js
return this.files + this.folders + (this.showSpaces ? this.spaces : 0)
```

Reply:
And more radical: isn't `this.spaces` always `0` if `this.showSpaces` evaluates to `false`? In that case you could just do

```js
return this.files + this.folders + this.spaces
```